spec/system/users/edit_user_spec.rb

```diff
@@ -22,15 +23,9 @@
     end
 
     scenario 'can change the user data' do
-      visit edit_user_registration_path
+      edit_user_page.visit_page.fill_in_with(new_email, password).submit
```

Comment:
I prefer
```ruby
edit_user_page.visit_page

edit_user_page.fill_in_with(new_email, password)

edit_user_page.submit
```

Reply:
This is just what I was trying to avoid 🤣 . Repeating this lines in every test could make it grow very quickly, but if you think it is more clear, I change it

Reply:
I think I like it. There could be tests navigating several pages and I like knowing on which page I'm currently at